app/assets/javascripts/leaflet.map.js

```diff
@@ -9,6 +14,43 @@ L.extend(L.LatLngBounds.prototype, {
   }
 });
 
+if (OSM.MAPTILER_KEY) {
+  maplibregl.setRTLTextPlugin(
+    "https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-rtl-text/v0.2.3/mapbox-gl-rtl-text.js",
```

Comment:
To avoid this external dependency on a Mapbox API, install the `@mapbox/mapbox-gl-rtl-text` package and have a build script copy node_modules/@mapbox/mapbox-gl-rtl-text/dist/mapbox-gl-rtl-text.js into app/assets/javascripts/. Then this becomes just `mapbox-gl-rtl-text.js`.

Reply:
Which build script do you mean? I added this to `package.json` but `docker compose build` fails because such path doesn't exist when building docker image.

```
  "scripts": {
    "postinstall": "cp node_modules/@mapbox/mapbox-gl-rtl-text/mapbox-gl-rtl-text.min.js app/assets/javascripts/mapbox-gl-rtl-text.min.js"
  },
```

Reply:
@1ec5 and @HolgerJeromin any help/comments on that, so that we could more forward?

Reply:
See https://github.com/openstreetmap/openstreetmap-website/pull/4042#discussion_r1966816765, sorry for the delay.